couchbase/README.md

```diff
@@ -46,25 +46,27 @@ Metadata associated with SignalFx's Couchbase integration can be found <a target
 
 ### USAGE
 
-Below are screen captures of dashboards created for this plugin by SignalFx, illustrating the metrics emitted by this plugin.
+Screen captures below of dashboards created for this plugin by SignalFx illustrate the metrics emitted by this plugin.
 
 For general reference on how to monitor Couchbase, see <a target="_blank" href="http://blog.couchbase.com/monitoring-couchbase-cluster">Couchbase Monitoring</a> and <a target="_blank" href="http://developer.couchbase.com/documentation/server/4.0/monitoring/monitoring-rest.html">Monitor using the REST API</a>.
 
 **Note on bucket metrics**
 
-This plugin emits some metrics about the bucket's performance across the cluster, and some metrics about the bucket's performance per node.
+This plugin emits metrics about bucket performance across the cluster, and also metrics about bucket performance per node.
 
-Metrics beginning with `gauge.bucket.basic.​*` and `gauge.bucket.quota.*`​ are reported once per cluster. All other bucket metrics (`gauge.bucket.*`) are reported by every node that hosts that bucket. In order to analyze bucket performance for the entire bucket, apply functions like Sum or Mean to group node-level metrics together by bucket.
+Metrics beginning with `gauge.bucket.basic.​*` and `gauge.bucket.quota.*`​ are reported once per cluster. All other bucket metrics (`gauge.bucket.*`) are reported by every node that hosts that bucket.
+
+To analyze performance for the entire bucket, you can group node-level metrics together by applying functions like Sum or Mean to them.
 
 **Monitoring a Couchbase cluster**
 
-On the Couchbase Nodes overview dashboard, you can see at a glance the status the nodes and buckets in a given cluster. Nodes in the cluster should be seeing balanced activity. Buckets in the cluster should each have adequate memory remaining.
+On the Couchbase Nodes overview dashboard, you can see at a glance the status of nodes and buckets in a given cluster. Nodes in the cluster should show balanced activity. Buckets in the cluster should each have adequate memory remaining.
 
 ![Couchbase - Nodes and buckets in a cluster](././img/nodes_and_buckets_snapshot.png)
 
 *This cluster's three nodes have roughly the same number of gets per second, and its two buckets have plenty of headroom.*
 
-This dashboard also includes a percentile distribution of CPU utilization per node, allowing quick identification of unusually hot nodes. This chart shows minimum, 10th percentile, median (50th percentile), 90th percentile, and maximum CPU utilization for each node in the cluster.
+This dashboard also includes a percentile distribution of CPU utilization per node, enabling quick identification of unusually hot nodes. This chart shows minimum, 10th percentile, median (50th percentile), 90th percentile, and maximum CPU utilization for each node in the cluster.
```

Comment:
is it obvious what a "hot node" is?

Reply:
It wasn't, but I added a few words (hot meaning > 80% utilization)